src/main/java/duke/logic/CreateMap.java

```diff
@@ -9,6 +9,9 @@
 import java.util.HashMap;
 import java.util.Map;
 
+/**
+ * Class which stores information in the map corresponding to bus-stops and train lines.
```

Comment:
Maybe use more of the [verb] [a/the] description style of header comment. Not very sure if that's the style he's looking for